src/components/AnalysisModal.js

```diff
@@ -101,13 +101,14 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
     const getEnvironmentView = () => Utils.switchCase(currentTool,
       [tools.Jupyter.label, renderComputeModal],
       [tools.RStudio.label, renderComputeModal],
-      [tools.galaxy.label, () => renderAppModal(GalaxyModalBase, tools.galaxy.label)],
-      [tools.cromwell.label, () => renderAppModal(CromwellModalBase, tools.cromwell.label)]
+      [tools.Galaxy.label, () => renderAppModal(GalaxyModalBase, tools.Galaxy.label)],
+      [tools.Cromwell.label, () => renderAppModal(CromwellModalBase, tools.Cromwell.label)]
     )
 
     const renderComputeModal = () => h(ComputeModalBase, {
       isOpen: currentTool === tools.Jupyter.label || currentTool === tools.RStudio.label,
       isAnalysisMode: true,
+      shouldHideCloseButton: true,
```

Comment:
I don't understand why we would want to hide a close button here. Which close button is this referring to?

Reply:
Ah, I think I understand what it is now. It's an _inner_ close button. So now, my question is, when would you ever want it to be true?

![image](https://user-images.githubusercontent.com/484484/160697639-2489d565-e6f4-462c-a98c-f90f300d6d66.png)

Reply:
It turns out all this code is related to the old cloud environment widget, which opens these inner "modals" outside of the parent dialog.